docs/index.rst

```diff
@@ -40,11 +40,11 @@ MLRun provides the following key benefits:
 
 - **Rapid deployment** of code to production pipelines
 - **Elastic scaling** of batch and real-time workloads
-- **Feature management** – ingestion, preparation, and monitoring
-- **Works anywhere** – your local IDE, multi-cloud, or on-prem
+- **Feature management** – ingestion, preparation and monitoring
+- **Works anywhere** – your local IDE, multi-cloud or on-prem
```

Comment:
Please revert this change. Our convention is to use a serial comma (although it's not currently evident because most of the MLRun docs were written by R&D or Product and haven't gone through a thorough doc review yet).